assets/src/edit-story/app/media/media3p/api/test/apiFetcher.js

```diff
@@ -280,4 +303,49 @@ describe('ApiFetcher', () => {
       ).rejects.toThrow(/Invalid order_by/);
     });
   });
+
+  describe('registerUsage', () => {
+    it('should perform a POST request', async () => {
+      mockFetch(VALID_REGISTER_USAGE_RESPONSE, {
+        requestPath: Paths.REGISTER_USAGE,
+        requestMethod: 'POST',
+      });
+
+      const result = await apiFetcher.registerUsage({ payload: PAYLOAD });
+      expect(result).not.toBeNull();
```

Comment:
result may not be null, but it could (or perhaps should) be 'undefined' as the caller does not expect a response, though an exception may be loggged.

Reply:
Done.